package.json

```
@@ -56,8 +56,8 @@
     "jest": "^27.2.3",
     "jest-puppeteer": "^4.1.1",
     "js-beautify": "^1.9.1",
-    "jstransformer-marked": "^1.0.3",
-    "jstransformer-nunjucks": "^0.5.0",
+    "jstransformer-marked": "1.0.3",
```

Approved, but are patch updates acceptable?

`~` Patch
`^` Minor

```suggestion
    "jstransformer-marked": "~1.0.3",
```

---

I ummed and ahhed about it, but you're probably right.